example/package.json

```diff
@@ -4,8 +4,9 @@
   "type": "module",
   "version": "0.0.0",
   "scripts": {
-    "dev": "convex dev --live-component-sources",
+    "dev:backend": "convex dev --live-component-sources",
```

Comment:
shouldn't this have `--typecheck-components` too?

Reply:
is that in the template now? i can add it